api/src/db/migrations/v151_add_land_to_environment_data.test.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  generatev150Business,
+  generatev150UserData,
+  generatev150WasteData,
+  generatev150WasteQuestionnaireData,
+} from "@db/migrations/v150_remove_needs_nexus_dba_name";
+import { migrate_v150_to_v151 } from "@db/migrations/v151_add_land_to_environment_data";
+
+describe("v151_add_land_to_environment_data", () => {
+  it("migrates existing environment data", () => {
+    const id = "biz-1";
+    const v150Business = generatev150Business({
+      environmentData: {
+        waste: generatev150WasteData({
+          questionnaireData: generatev150WasteQuestionnaireData({
+            compostWaste: true,
+            constructionDebris: true,
+          }),
+          submitted: true,
+        }),
+      },
+      id,
+    });
+    const v150User = generatev150UserData({
+      businesses: { "biz-1": v150Business },
+    });
+
+    const v151User = migrate_v150_to_v151(v150User);
+
+    expect(v151User.businesses[id].environmentData?.waste?.questionnaireData).toEqual({
+      hazardousMedicalWaste: false,
+      constructionDebris: true,
+      compostWaste: true,
+      treatProcessWaste: false,
+      noWaste: false,
+    });
+    expect(v151User.businesses[id].environmentData?.waste?.submitted).toEqual(true);
+    expect(v151User.businesses[id].environmentData?.land).toBeUndefined();
+  });
```

Comment:
dust - might be good to have a second test, that covers a user with values false?

Or at least add some more assertions here that before this change those new field we added were undefined? (a bit nit picky on my end)

Reply:
good call, I added more verbose and thorough tests!